Makefile

```diff
@@ -35,7 +35,7 @@ iso_creation: init
 	dd if=$(BUILD_DIR)/$(ISO_NAME).iso bs=2k skip=1 >> ${BUILD_DIR}/tmp.iso
 	mv -f ${BUILD_DIR}/tmp.iso $(BUILD_DIR)/$(ISO_NAME).iso
 
-.PHONY: check
+.PHONY: check_env
```

Comment:
This is their execution:

#### make check
```
[vagrant@localhost minishift-centos-iso]$ make check
make: Nothing to be done for `check'.
```

#### make check_env
```
[vagrant@localhost minishift-centos-iso]$ make check_env
if test "" = ""; then \
	echo "rhel_tree_url is undefined, Please check README"; \
	exit 1; \
elif test "" = ""; then \
	echo "base_repo_url is undefined, Please check README"; \
	exit 1; \
elif test "" = ""; then \
	echo "updates_repo_url is undefined, Please check README"; \
	exit 1; \
fi
rhel_tree_url is undefined, Please check README
```

I am bit wondering why there is a name needed as target which has nothing to do